.github/workflows/rpk-build.yml

```diff
@@ -10,7 +10,8 @@
 name: Go
 on:
   push:
-    tags: 'release-*'
+    tags: 
+    - 'v2**'
```

Comment:
`v[0-9]+\.[0-9]+\.[0-9]+` ?

Reply:
I left it like this in case we wanted to go with something like `v20.12.2-rc2` or a format like that.